.github/workflows/build.yml

```diff
@@ -0,0 +1,37 @@
+---
+on: [push, pull_request]
+name: Build
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
```

Comment:
i don't think this job is required because we already build our code in other workflows